### PR TITLE
Editting custom element

### PR DESCRIPTION
**Pull request by MumenTayyem**

### CODE REVIEW

**Comment by MumenTayyem** [comment]
ci:test:sf

**Comment by MumenTayyem** [comment]
ci:test:sf

**Comment by ethib137** [comment]
ci:test:sf

**Review by ethib137 — CHANGES_REQUESTED** [review]
Can you add these changes to your new PR? I'll finish reviewing the new PR.

File: workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/components/TicketPreview.tsx [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+import {assignTicketToMe} from '../services/tickets';
+export const SPRITEMAP =
+	(Liferay as any).ThemeDisplay.getPathThemeImages() + '/clay/icons.svg';
+
+const usersIconsMap: {[key: string]: string} = {
+	'': icons.NONE,
+	'M2H8_ADMIN_ADAM': icons.ADAM,
+	'M2H8_CUSTOMER_CARL': icons.CARL,
+	'M2H8_SUPPORT_SAM': icons.SAM,
+};
```
[the remark this answers is not in the document]
Comment (MumenTayyem) [reply workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/components/TicketPreview.tsx:30]: Hey @ethib137, these users are created using the batch framework, unfortunately, users images are not supported yet.. so I had to add the images into the assets folder inside the custom element and map them based on the user ERC.

File: workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/components/StatusColumn.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+			ref={setNodeRef}
+		>
+			<h6>{status.name}</h6>
+
+			{status.relatedTickets?.length === 0 && (
+				<p>No tickets are available.</p>
+			)}
+
+			{status.relatedTickets?.length > 0 &&
+				status.relatedTickets.map((t: any) => (
```
[the remark this answers is not in the document]
Comment (MumenTayyem) [reply workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/components/StatusColumn.tsx:34]: Yes, I'll fix this.

File: workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/components/StatusColumn.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+	queryClient,
+	status,
+}) => {
+	const {setNodeRef} = useDroppable({id: status.key + '_droppable'});
+
+	return (
+		<div
+			className="status-col"
+			id={status.key + '_droppable'}
+			key={status.key + '_droppable'}
```
[the remark this answers is not in the document]
Comment (MumenTayyem) [reply workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/components/StatusColumn.tsx:24]: Yes, correct.

File: workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/components/StatusColumn.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+			ref={setNodeRef}
+		>
+			<h6>{status.name}</h6>
```
[the remark this answers is not in the document]
Comment (MumenTayyem) [reply workspaces/liferay-ticket-workspace/client-extensions/liferay-ticket-custom-element/src/components/StatusColumn.tsx:27]: I changed it to the paragraph tag

**Pull request closed without merging**